### ISSUE TITLE
Problems when using flatlaf 0.33 on Apache NetBeans 11.3, JDK14 - Ubuntu 20.04

### ISSUE THREAD

**Issue by UltimateYZ** [JFormDesigner/FlatLaf#93]
I've been using flatlaf for quite a time on linux, but since the new update (0.33), I'm having problems with applying the look and feel in my project. I already tried the method - FlatLightLaf.install(); or try {
    UIManager.setLookAndFeel( new FlatLightLaf() );
} catch( Exception ex ) {
    System.err.println( "Failed to initialize LaF" );
}
 But it says: cannot find symbol (or class), I've already added the dependencies in my class path ( I'm using Ant build ). I don't know what to do any more.
 
![Screenshot from 2020-05-02 12-46-35](https://user-images.githubusercontent.com/55871338/80862153-120a1600-8c73-11ea-85c7-b9ab9d8dcdbd.png)

**Comment by DevCharly** [comment]
Yes, this is a known issue in NetBeans editor, which is fixed since yesterday for NetBeans 12 with PR https://github.com/apache/netbeans/pull/2115.

Maybe it is already included in latest daily build of NB 12.
Beta 4 of NB 12 should be available in some days.
Final NB 12 in about 2 weeks.

Anyway, it seems that only the editor shows an error.
I can still compile and run the project without errors.

So either simply ignore the error for now and update to NB 12 later, 
or go back to FlatLaf 0.32 (not recommended),
or pass a string to `UIManager.setLookAndFeel`. E.g.

~~~java
UIManager.setLookAndFeel("com.formdev.flatlaf.FlatLightLaf");
~~~

**Comment by DevCharly** [comment]
Just released version 0.34 that includes a workaround for this issue (reorder files in JAR).
So FlatLaf 0.34 can be used in NetBeans 11.3 (and older) without errors.
